client.go

```diff
@@ -73,14 +73,12 @@ func (client *TwilioClient) post(values url.Values, uri string) ([]byte, error)
 }
 
 func (client *TwilioClient) get(queryParams url.Values, uri string) ([]byte, error) {
-	var params *strings.Reader
-
 	if queryParams == nil {
 		queryParams = url.Values{}
 	}
 
-	params = strings.NewReader(queryParams.Encode())
-	req, err := http.NewRequest("GET", client.buildUri(uri), params)
```

Comment:
Passing the filters to AvaliablePhoneNumbers as body of GET didn't work (nor would I really expect it to)

There are two other places in master where a non-null can be passed as the body:
here: https://github.com/carlosdp/twiliogo/blob/0a85fda8d6e854bdbefa188be21a11bf37c8a6f4/incoming_phone_number_list.go#L34

and here:
https://github.com/carlosdp/twiliogo/blob/0a85fda8d6e854bdbefa188be21a11bf37c8a6f4/message_list.go#L34

neither currently are unit tested (with params) and I would be surprised if they worked before, but you should maybe test that before merging this?